src/qililab/platform/platform.py

```diff
@@ -576,11 +576,34 @@ def execute_qprogram(  # pylint: disable=too-many-locals
     ) -> QProgramResults:
         """Execute a QProgram using the platform instruments.
 
+        |
+
+        **The execution is done in the following steps:**
+
+        1. Compile the QProgram.
+        2. Upload the sequences to the instruments.
+        3. Execute the sequences.
+        4. Acquire the results.
+        5. Reset the instrument settings.
+
+        |
+
+        **The execution is done in the following way:**
+
+        - If all the buses in the QProgram are associated with Qblox modules, the execution is done using the QbloxCompiler.
+        - If all the buses in the QProgram are associated with Quantum Machines Clusters, the execution is done using the QuantumMachinesCompiler.
+        - If the buses in the QProgram are associated with a mixture of Qblox modules and Quantum Machines Clusters, the execution is not supported.
+
         Args:
             qprogram (QProgram): The QProgram to execute.
+            bus_mapping (dict[str, str], optional): A dictionary mapping the buses in the QProgram to the buses in the platform.
+                The keys are the buses in the QProgram, and the values are the buses in the platform. Defaults to None.
+            calibration (Calibration, optional): The calibration to use for the execution. Defaults to None.
```

Comment:
This exaplanation is too broad, this calibration makes reference to the yaml file containing information of the experiment (i.e. pulse duration, amp...)

I would say something like:
calibration (Calibration, optional): Calibration file containing information of te experiment in the YAML format. Defaults to None.

Reply:
Cool! I'll change it, thanks!